pl_examples/hpu_examples/simple_mnist/ops_bf16_mnist.txt

```diff
@@ -0,0 +1,2 @@
+linear
+relu
```

Comment:
who maintains this ops list over time? at present it seems likely it will fall out of sync.
can there be some comments here for what this file is intended to include?

Reply:
These are just optional inputs. If the user doesn't want to override any specific op, user need not use this parameter.